core/domain/exp_jobs_one_off.py

```diff
@@ -540,3 +540,18 @@ def reduce(key, values):
             yield (key, len(values))
         else:
             yield (key, values)
+
+
+class DeleteStateIdMappingModelsOneOffJobs(jobs.BaseMapReduceOneOffJobManager):
+    """One-off job for wiping out state ID mapping models."""
+    @classmethod
+    def entity_classes_to_map_over(cls):
+        return [exp_models.StateIdMappingModel]
+
+    @staticmethod
+    def map(item):
+        item.delete()
+
```

Comment:
Do we have to delete the Memcache as well (not sure)? or we are going to do it manually?

Reply:
No need, we do it manually before running jobs.